### PR TITLE
cleaned up some of the React boiler plate

### PR DESCRIPTION
**Pull request by Shreshth3**
-Deleted some (but certainly not all) unnecessary files
-Converted some files from .tsx to .jsx
-Cleaned out App component and index.tsx
-Renamed server folder to be kafkaApplicationSimulator
-Deleted old and unused files from kafkaApplicationSimulator

### CODE REVIEW

File: app/app.global.css [hunk 1]
```diff
@@ -1,47 +1,5 @@
...
-  color: white;
-  opacity: 0.75;
-  text-decoration: none;
-}
-
-a:hover {
-  opacity: 1;
-  text-decoration: none;
-  cursor: pointer;
+* {
```
Comment (Jbaren01) [inline app/app.global.css:1]: good work here copying what I sent you.

File: app/app.html [hunk 2]
```diff
@@ -40,7 +40,7 @@
         if (scripts.length) {
           document.write(
             scripts
-              .map(script => `<script defer src="${script}"><\/script>`)
+              .map((script) => `<script defer src="${script}"><\/script>`)
```
Comment (Jbaren01) [inline app/app.html:43]: why did you delete then copy same thing? just a mistake?

File: kafkaApplicationSimulator/controllers/kafkaController.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const { Kafka } = require('kafkajs');
+const { calculateThroughput } = require('kafkaq');
```
Comment (Jbaren01) [inline kafkaApplicationSimulator/controllers/kafkaController.js]: where is this coming from?

**Review by Jbaren01 — COMMENTED** [review]
Comments for Shreshth to review

File: app/app.html (hunk 2 above)
Comment (Shreshth3) [reply app/app.html:43]: eslint added parentheses around the script parameter

File: kafkaApplicationSimulator/controllers/kafkaController.js (hunk 3 above)
Comment (Shreshth3) [reply kafkaApplicationSimulator/controllers/kafkaController.js]: my bad, that was a fake thing i was using to explain to nicole. i will delete this

Comment (Shreshth3) [reply kafkaApplicationSimulator/controllers/kafkaController.js]: done

**Review by Jbaren01 — APPROVED** [review]
This is approved
